pkg/subctl/cmd/show_connections.go

```diff
@@ -58,10 +59,12 @@ func getConnectionsStatus(submariner *v1alpha1.Submariner) []connectionStatus {
 		for _, connection := range gateway.Connections {
 			subnets := strings.Join(connection.Endpoint.Subnets, ", ")
 
+			ip, nat := remoteIPAndNATForConnection(&connection)
```

Comment:
The linter is complaining about this because taking the address of a range iterator is not good. To fix this, add a loop index and use `&gateway.Connections[i]` (and use `gateway.Connections[i]` throughout instead of `connection`).

Reply:
Oh I thought I force pushed to fix it but I must have been too sleepy 

thanks for the explanation @skitt

Reply:
I just passed the connection, without the pointer, so there's no need for the index, linter is passing now :)